### ISSUE TITLE
Tutorial and blender file inconsitent in terms of shader config

### ISSUE THREAD

**Issue by frippe75** [PCB-Arts/stylized-blender-setup#4]
Just think the shader setup differs a lot from the tutorial part.
At least from what I can tell.
![Screenshot from 2022-09-11 19-04-31](https://user-images.githubusercontent.com/7848204/189540006-a9bdfcf7-4b9b-48ec-a273-000158d8bf4a.png)

Drill holes are not there as an example

**Comment by flonkler** [comment]
That's correct. This repository hasn't been updated in a while. The tutorial shows a more recent and better structured version of the shader. However, end result is almost the same. If you want to create your own renders, I'll suggest that you follow the instructions of the tutorials.
